keras/ops/nn.py

```diff
@@ -1840,3 +1840,57 @@ def ctc_loss(target, output, target_length, output_length, mask_index=0):
     return backend.nn.ctc_loss(
         target, output, target_length, output_length, mask_index
     )
+
+
+class Normalize(Operation):
+    def __init__(self, order=2.0, axis=1, eps=1e-12):
+        super().__init__()
+        self.p = order
+        self.axis = axis
+        self.eps = eps
+
+    def compute_output_spec(self, x):
+        return KerasTensor(shape=x.shape)
+
+    def call(self, x):
+        return _normalize(x, self.order, self.axis, self.eps)
+
+
+@keras_export(
+    [
+        "keras.ops.normalize",
+        "keras.ops.nn.normalize",
+    ]
+)
+def normalize(x, order=2.0, axis=1, eps=1e-12):
+    """Perform Lp normalization of a tensor over the specified axis.
+        `v' = v / max(||v||_p, epsilon)`
```

Comment:
Use a code block and use Python syntax

Reply:
Do you mean like this?
```
"""Perform Lp normalization of a tensor over the specified axis.
```python
        v' = v / max(||v||_p, epsilon)
```
Noticed `sigmoid()` uses following formatting,
https://github.com/keras-team/keras/blob/5bc8488c0ea3f43c70c70ebca919093cd56066eb/keras/activations/activations.py#L319-L322

Reply:
You can do:

```
It is defined as: `normalize(x) = x / max(norm(x), epsilon)`.
```

Reply:
Thanks. I've updated the code.